code/game/machinery/autolathe.dm

```diff
@@ -164,7 +164,10 @@
 			var/multiplier = text2num(href_list["multiplier"])
 			var/max_multiplier = min(50, being_built.materials[MAT_METAL] ?round(materials.amount(MAT_METAL)/being_built.materials[MAT_METAL]):INFINITY,being_built.materials[MAT_GLASS]?round(materials.amount(MAT_GLASS)/being_built.materials[MAT_GLASS]):INFINITY)
 			var/is_stack = ispath(being_built.build_path, /obj/item/stack)
-
+			
+			if (ispath(being_built.build_path, /obj/item/stack/cable_coil))
+				max_multiplier = min(30, max_multiplier)

```

Comment:
Putting a specific ispath() check onto the generic stack code isn't ideal. Could you make this a var in the /datum/design? `stack_max` or something, make it default to 50 and override it for cable coils.